src/style.css

```diff
@@ -0,0 +1,24 @@
+@import "./card.css";
+@import "./stars.css";
+
+:root {
+  --primary-text-color: #060b35;
+  --secondary-text-color: #616070;
+  --background-color: #fff;
+  --accent-color: #00acdc;
+}
+
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+  margin: 0;
```

Comment:
Do not reset margins using * selector.
With lot's of elements on the page, it may be performance heavy.
Use tags.

Border-box, however, is fine to use with *